charts/k8ssandra/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: k8ssandra
 description: |
   Provisions and configures an instance of the entire K8ssandra stack. This includes Apache Cassandra, Stargate, Reaper, Medusa, Prometheus, and Grafana.
 type: application
-version: 0.55.0
+version: 0.56.0
```

Comment:
You'll need to bump to 57.

Reply:
Yeah we're definitely at 56 already, strange that it doesn't seem to show that here, hmm.

https://github.com/k8ssandra/k8ssandra/blob/df1f5f44925b247a483014176995d4d691ada417/charts/k8ssandra/Chart.yaml#L6